setup.cfg

```diff
@@ -22,9 +22,9 @@ install_requires =
     #       tests for this package.
 
     # AIX requires fixes contained in numpy 1.16
-    numpy==1.16.0; python_version=='3.5' and platform_system=='AIX'
-    numpy==1.16.0; python_version=='3.6' and platform_system=='AIX'
-    numpy==1.16.0; python_version=='3.7' and platform_system=='AIX'
+    numpy==1.16.0; python_version=='3.5' and platform_system=='AIX' and (platform_machine!='aarch64' or platform_python_implementation == 'PyPy')
+    numpy==1.16.0; python_version=='3.6' and platform_system=='AIX' and (platform_machine=='loongarch64' or platform_machine!='aarch64' and platform_python_implementation != 'PyPy')
+    numpy==1.16.0; python_version=='3.7' and platform_system=='AIX' and (platform_machine=='loongarch64' or platform_machine!='aarch64' and platform_python_implementation != 'PyPy')
```

Comment:
I don't fully understand yet why this condition is necessary, but if you're mixing `and` and `or`, please add brackets, so we don't have to do operator precedence in our heads.